Explorer/Assets/DCL/Chat/History/ChatHistoryStorage.cs

```diff
@@ -201,16 +204,21 @@ public async UniTask InitializeChannelWithMessagesAsync(ChatChannel.ChannelId ch
                 (channelFile.Content != null && channelFile.Content.CanRead))
             {
                 ReportHub.LogWarning(reportData, $"Initialization canceled. The file does not exist or the channel was already initialized: " + channelId.Id);
-                return;
+                return false;
             }
 
             messagesBuffer.Clear();
             await ReadMessagesFromFileAsync(channelId, messagesBuffer);
 
-            if (messagesBuffer.Count > 0)
+            bool loadedAny = messagesBuffer.Count > 0;
```

Comment:
So if there were no messages, the channel can't be initialized? (I'm assuming that's what the return value represents, if it could be initialized or not, since the function comment doesn't explain what the return value means)

Channels cannot be initialized if there are not messages for them?